### DISCUSSION TITLE
fMRI pre proc data "BOLD" avaliable?

### DISCUSSION THREAD

**Discussion by SonaLily** [aramis-lab/clinicadl#341] [Q&A]
I was wondering if clinca/clinca-dl has preprocessed fMRI, BOLD, data or if there are any known places that provide it? Thank you in advance

**Comment by NicolasGensollen** [comment]
Hi @SonaLily 
I you're looking for preprocessed datasets, you can check out [Nilearn](https://github.com/nilearn/nilearn) which provides some fetchers to well known fMRI datasets (take a look at the datasets module and relevant examples).
Hope that helps !

**Reply by SonaLily** [reply]
Thanks a lot for your response. I did fetch preprocessed fmri data such as ASD data via Nilearn. I now need preprocessed Oasis3 fmri data for GNN.  Is the BOLD data of Oasis3 preprocessed fmri? if it is not preprocessed fmri, does clinica/clinicadl or any other tools have pipeline to preprocess Oasis fmri? if not, how to preprocess Oasis fmri data myself? Thanks a lot

**Reply by SonaLily** [reply]
Basically where can I get preprocessed fmri Alzheimer data? if no preprocessed fmri Alzheimer data are available, how to preprocess fmri Alzheimer data myself? Thanks
